sql_generators/usage_reporting/templates/usage_reporting_clients_daily_v1.query.sql

```diff
@@ -0,0 +1,66 @@
+    metrics.string.usage_app_channel AS normalized_channel,
+    normalized_country_code,
+    -- normalized_os,
+    metrics.string.usage_os AS normalized_os,
+    -- normalized_os_version,
+    metrics.string.usage_os_version AS normalized_os_version,
+    metrics.string.usage_app_display_version AS app_display_version,
+    metrics.string.usage_app_build AS app_build,
+    {% if has_distribution_id %}
+    metrics.string.metrics_distribution_id AS distribution_id,
+    {% else %}
+    CAST(NULL AS STRING) AS distribution_id,
+    {% endif %}
+    metrics.boolean.metrics_default_browser AS is_default_browser,
+    metrics.string.usage_reason AS reason,
+    {% if "_desktop" in app_name %}
+    COALESCE(metrics.counter.browser_engagement_uri_count, 0) AS browser_engagement_uri_count,
+    COALESCE(metrics.counter.browser_engagement_active_ticks, 0) AS browser_engagement_active_ticks,
+    {% endif %}
+    CAST(NULL AS BOOLEAN) AS is_active,  -- Eventually is_active value will come from the client.
+    metrics.timespan.usage_duration.value AS duration,
+    -- client_info.locale, -- missing in the ping
+
+  FROM
+    `{{ project_id }}.{{ channel_dataset }}_stable.{{ usage_reporting_stable_table_name }}`
+  WHERE
+    -- {% raw %}
+    -- {% if is_init() %}
+    -- DATE(submission_timestamp) >= '2024-10-10'
+    -- {% else %}
+    DATE(submission_timestamp) = @submission_date
+    -- {% endif %}
+    -- {% endraw %}
+    AND metrics.uuid.usage_profile_id IS NOT NULL
+)
+SELECT
+  submission_date,
+  usage_profile_id,
+  udf.mode_last(ARRAY_AGG(first_run_date IGNORE NULLS ORDER BY submission_timestamp ASC)) AS first_run_date,
+  udf.mode_last(ARRAY_AGG(normalized_channel IGNORE NULLS ORDER BY submission_timestamp ASC)) AS normalized_channel,
+  udf.mode_last(ARRAY_AGG(normalized_country_code IGNORE NULLS ORDER BY submission_timestamp ASC)) AS normalized_country_code,
+  udf.mode_last(ARRAY_AGG(normalized_os IGNORE NULLS ORDER BY submission_timestamp ASC)) AS normalized_os,
+  udf.mode_last(ARRAY_AGG(normalized_os_version IGNORE NULLS ORDER BY submission_timestamp ASC)) AS normalized_os_version,
+  udf.mode_last(ARRAY_AGG(app_display_version IGNORE NULLS ORDER BY submission_timestamp ASC)) AS app_display_version,
+  udf.mode_last(ARRAY_AGG(app_build IGNORE NULLS ORDER BY submission_timestamp ASC)) AS app_build,
+  udf.mode_last(ARRAY_AGG(distribution_id IGNORE NULLS ORDER BY submission_timestamp ASC)) AS distribution_id,
+  udf.mode_last(ARRAY_AGG(is_default_browser IGNORE NULLS ORDER BY submission_timestamp ASC)) AS is_default_browser,
+  udf.mode_last(ARRAY_AGG(reason IGNORE NULLS ORDER BY submission_timestamp ASC)) AS reason,
+  {% if "_desktop" in app_name %}
+  COALESCE(LOGICAL_OR(is_active), SUM(browser_engagement_uri_count) > 0 AND SUM(browser_engagement_active_ticks) > 0, FALSE) AS is_active,
+  {% else %}
+  COALESCE(LOGICAL_OR(is_active), SUM(IF(duration BETWEEN 0 AND 100000, duration, 0)) > 0, False) AS is_active,
```

Comment:
I think we might need a `COALESCE(..., 0)` on `duration`, `browser_engagement_uri_count` and `browser_engagement_active_ticks` here. They can be null (duration is often null since its only sent on inactive pings)